.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "lemmy-translations"]
 	path = lemmy-translations
-	url = https://github.com/lemmynet/lemmy-translations
-	branch = main
+	url = https://github.com/Ategon/Lemmy-Translations
```

Comment:
do you want to keep this pointed at your fork or update to the programming.dev one?